bindings/rust/s2n-tls/src/testing/resumption.rs

```diff
@@ -50,6 +50,15 @@ mod tests {
     const KEY: [u8; 16] = [0; 16];
     const KEYNAME: [u8; 3] = [1, 3, 4];
 
+    fn validate_session_ticket(c: &mut Connection) -> Result<(), Box<dyn Error>> {
```

Comment:
Nit: we generally avoid one letter variables, could we use `conn` instead?

Reply:
sure